conda_smithy/feedstock_io.py

```diff
@@ -75,10 +75,20 @@ def remove_file(filename):
 
 
 def copy_file(src, dst):
-    with io.open(src, "r", encoding="utf-8") as fh_src:
-        with io.open(dst, "w", encoding="utf-8", newline="\n") as fh_dst:
-            for line in fh_src:
-                fh_dst.write(line)
+    """
+    Tried to copy utf-8 text files line-by-line to avoid
+    getting CRLF characters added on Windows.
```

Comment:
Tried the development version from this branch on a local feedstock and it seemed to work fine: only a `.sh` file was changed, and no CRLF changes at all. 👍